spec/integration/directly_contains_one_association_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'spec_helper'
+
+describe ActiveFedora::Base do
+  before do
```

Comment:
I'd like to see a test here that uses `directly_contains` and a class that extends `ActiveFedora::Base` (instead of just `ActiveFedora::File`)